apps/docs/analytics/query-examples.mdx

```diff
@@ -0,0 +1,762 @@
+---
+title: Query Examples
+description: "Common SQL patterns for analytics and billing"
+---
+
+This guide provides SQL query examples for common analytics scenarios covering all the use cases from the legacy API and more. All examples use ClickHouse SQL syntax and work with the `/v2/analytics.getVerifications` endpoint.
+
+## Using Queries in API Requests
+
+When making API requests, you need to format the SQL query as a JSON string on a single line. Here's how:
+
+**Readable format (for documentation):**
+
+```sql
+SELECT COUNT(*) as total
+FROM key_verifications_v1
+WHERE time >= now() - INTERVAL 7 DAY
```

Comment:
For longer queries converting SQL to single line might take some time. Maybe we can give them a link or some simple way to quickly turn SQL into single line. I'd love to have that DX as a customer